helmcharts/stable/README.md

```diff
@@ -25,26 +25,28 @@ $ kubectl create namespace eclipse-che
 
 Install `Eclipse Che Operator` by following instructions in top right button `Install`.
 
-A new pod che-operator is created in `eclipse-che` namespace
+A new pod `che-operator` is created in `eclipse-che` namespace:
 
 ```bash
 $ kubectl get pods -n eclipse-che
 eclipse-che   che-operator-554c564476-fl98z                           1/1     Running   0          13s
 ```
 
-Click `CRDS` button, select `CheCluster` template and copy 
-custom resource Eclipse Che to file org.eclipse.che_v1_checluster.yaml.
-Set up valid actual ingress domain `k8s.domain`.
-Apply CR:
+Click `CRDS` button, select `CheCluster` template and save Eclipse Che custom resource as a file `org.eclipse.che_v1_checluster.yaml`.
+Update Eclipse Che custom resource to set a valid ingress domain for a Kubernetes cluster:
+
+```bash
+yq -riY  '.spec.k8s.ingressDomain = "<DOMAIN>"' org.eclipse.che_v1_checluster.yaml
```

Comment:
Maybe here is we have to use kubectl. I'm not sure that users notified earlier to install yq, and I don't think that's a good requirement.